apps/editing-toolkit/editing-toolkit-plugin/global-styles/src/global-styles-sidebar.js

```diff
@@ -51,6 +55,17 @@ const PanelActionButtons = ( {
 	</div>
 );
 
+function maybeOpenSidebar() {
+	const openSidebar = getQueryArg( window.location.href, 'openSidebar' );
+	if ( 'global-styles' === openSidebar ) {
+		// Ideally this tracks event would be paired with the <a> tag at the source: class-global-styles-fonts-message-control.php
+		// instead of at the destination of the <a> tag, which is here
+		// I wasn't sure how to record a tracks from that file.
+		recordTracksEvent( 'calypso_global_styles_customizer_link_clicked' );
```

Comment:
Yes, we can record Tracks events server-side too: PCYsg-4KT-p2

Reply:
Thanks. Tracks events are being reported from javascript functions now, and those functions are constructed serverside.